lib/stat_tracker.rb

```diff
@@ -85,37 +85,123 @@ def average_scores_by_team
   end
 
   def best_offense
-    highest_avg_score = average_scores_by_team.max_by do |team, avg_score|
+    highest_avg_score = average_scores_by_team.max_by do |_team, avg_score|
       avg_score
     end
     find_team_by_id(highest_avg_score.first).team_name
   end
 
   def worst_offense
-    lowest_avg_score = average_scores_by_team.min_by do |team, avg_score|
+    lowest_avg_score = average_scores_by_team.min_by do |_team, avg_score|
```

Comment:
@rrabinovitch what does that `_` before team in the block variable do?

Reply:
it's an indicator that that block variable isn't actually utilized in the code block

Reply:
so it doesn't actually _do_ anything (as far as I understand); rather it seems to be a conventional thing